qa/1781

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# PCP QA Test No. 1781
+# Testing pmdahdb
+#
+# Copyright (c) 2025 Sam Feifer.  All Rights Reserved.
+#
+
+if [ $# -eq 0 ]
+then
+    seq=`basename $0`
+    echo "QA output created by $seq"
+else
+    # use $seq from caller, unless not set
+    [ -n "$seq" ] || seq=`basename $0`
+    echo "QA output created by `basename $0` $*"
+fi
+
```

Comment:
No need for this conditional section BTW, no valgrind use here so we can simplify this section to just lines 10 & 11.